### PR TITLE
Rename own languages

### PR DESCRIPTION
**Pull request by dslmeinte**
This is [issue #195 on specification](https://github.com/LionWeb-io/specification/issues/195).

### CODE REVIEW

File: packages/core/src/m3/builtins.ts [hunk 1]
```diff
@@ -1,18 +1,23 @@
 import {LanguageFactory} from "./factory.js"
 import {lioncoreBuiltinsKey, Property} from "./types.js"
-import {concatenator} from "../utils/string-mapping.js"
+import {StringsMapper} from "../utils/string-mapping.js"
```
Comment (enikao) [inline packages/core/src/m3/builtins.ts:3]: Should this be here?

**Review by enikao — APPROVED** [review]
Only checked for validity, not omissions.

File: packages/core/src/m3/builtins.ts (hunk 1 above)
Comment (dslmeinte) [reply packages/core/src/m3/builtins.ts:3]: What do you mean? The type of the `lioncoreBuiltinsIdAndKeyGenerator` constant is `StringsMapper`. (I could define that constant without explicitly stating that type, though.)

Comment (enikao) [reply packages/core/src/m3/builtins.ts:3]: I meant: "Should this be in this PR?"
Not really important, it's the only non-pure renaming change.

Comment (dslmeinte) [reply packages/core/src/m3/builtins.ts:3]: Ah yeah, now I get it :) Good question. This particular change comes on the back of cleaning up the whole ID and key generation around LionCore M3, which was a bit too “magical”.

**Pull request merged**